packages/wrangler/CHANGELOG.md

```diff
@@ -72,7 +84,7 @@
 
   ```jsx
   import SomeDependency from "some-dependency.js";
-  addEventListener("fetch", (event) => {
+  addEventListener("fetch", event => {
```

Comment:
so changesets is using an older version of prettier, which removes the parentheses, which then fails our check (https://github.com/changesets/changesets/issues/616). I think a fix for us is to simply not run prettier --check on this one file, or we'll keep failing releases.